functests/utils/profiles/profiles.go

```diff
@@ -47,3 +47,21 @@ func GetConditionMessage(nodeLabels map[string]string, conditionType v1.Conditio
 	}
 	return ""
 }
+
+// GetAllProfiles gets all the exiting profiles in the cluster
+func GetAllProfiles() (*performancev1alpha1.PerformanceProfileList, error) {
```

Comment:
The package is already `profiles`, this reads out as `profiles.GetAllProfiles`.
`profiles.GetAll` reads the same and sounds better, or even `profiles.All()`